### PR TITLE
Basic setup of Website Calender in Remix

### PR DESCRIPTION
**Pull request by harshith-venkatesh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+npm run build
+```
+
+Then run the app in production mode:
+
+```sh
+npm start
+```
+
+Now you'll need to pick a host to deploy it to.
```
Comment (rohan09-raj) [inline README.md:29]: This sentence seems incomplete to me. How about this ??
```suggestion
Now you'll need to pick a host to deploy the build/code.
```

File: app/routes/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+export default function Index() {
+  return (
+    <div style={{ fontFamily: "system-ui, sans-serif", lineHeight: "1.4" }}>
```
Comment (rohan09-raj) [inline app/routes/index.tsx:3]: Can we avoid inline css ??

**Review by rohan09-raj — COMMENTED** [review]
Nice initial setup 💯

File: app/routes/index.tsx (hunk 2 above)
Comment (harshith-venkatesh) [reply app/routes/index.tsx]: addressed

File: tailwind.config.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  content: ["./app/**/*.{ts,tsx,jsx,js}"],
+  theme: {
+    color: {
+      resolutionBlue:	'#041484',
+      razzmatazz: '#E30464',
+      feijoa: '#96D882',
+      fern:	'#6DB470'
```
Comment (rohan09-raj) [inline tailwind.config.js:8]: Can we have 1 tab spacing here as above please ??

**Review by rohan09-raj — CHANGES_REQUESTED** [review]
Rest all looks good to me 💯

File: tailwind.config.js (hunk 3 above)
Comment (harshith-venkatesh) [reply tailwind.config.js]: resolved

**Review by rohan09-raj — APPROVED** [review]
Looks good to me 🔥

**Pull request merged**